eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/PageConverter.php

```diff
@@ -228,7 +227,19 @@ protected function generateBlockXmlString(Parts\Block $block, DOMDocument $dom)
                     $blockNode->appendChild($node);
 
                     foreach ($attrValue as $arrayItemKey => $arrayItemValue) {
-                        $this->addNewXmlElement($dom, $blockNode, $arrayItemKey, $arrayItemValue);
+                        $this->addNewXmlElement($dom, $node, $arrayItemKey, $arrayItemValue);
```

Comment:
It should have been exoplained in the PR description why things are changed like thay are, but this can be seen in the issue. Basically attributes of these two tags should be added as childs to the `rotation`/`customAttributes` tag, and not the `block` tag.